dev-preview/README.md

```diff
@@ -10,27 +12,9 @@ Features on Development Preview
 - [Ansible Collection \& Inventory Plugin](#ansible-collection--inventory-plugin)
   - [Installation](#installation)
   - [Usage](#usage)
-- [Search-v2 - Odyssey](#search-v2---odyssey)
-  - [Installation](#installation-1)
-  - [Usage](#usage-1)
-- [Configurable Collection in Search](#configurable-collection-in-search)
-  - [Installation](#installation-2)
-  - [Usage](#usage-2)
 - [Dynamic Metric Collection (Custom Metrics Collection)](#dynamic-metric-collection-custom-metrics-collection)
```

Comment:
did you confirm with Joydeep? this has been in the product for SNO specifically, since 3-4 releases now. It should likely be considered GA by now.

Reply:
This is tech preview now in the product, we don't specifically call it SNO since ACM 2.7 the documentation started a section on Search Configurable Collection.